python/mxnet/_numpy_op_doc.py

```diff
@@ -176,3 +176,229 @@ def _np_cumsum(a, axis=None, dtype=None, out=None):
+    array([2, 3])
+    >>> np.amax(a, axis=1)   # Maxima along the second axis
+    array([1, 3])
+    >>> np.amax(a, axis=1, keepdims=True)
+    array([[1],
+          [3]])
+
+    Notes
+    -----
+    This function differs to the original `numpy.amax
+    <https://docs.scipy.org/doc/numpy/reference/generated/numpy.amax.html>`_ in
+    the following aspects:
+
+    - The default value type is `float32` instead of `float64` in numpy.
+    - `a` only supports ndarray.
+    - `axis` doe snot support negative value.
+    - `initial` is not supported.
+    """
+    pass
+
+
+def _np_squeeze(a, axis=None):
+    r"""
+    Remove single-dimensional entries from the shape of an array.
+
+    Parameters
+    ----------
+    a : ndarray
+        Input data.
+
+    axis : None or int or tuple of ints, optional
+
+    Returns
+    -------
+    squeezed : ndarray
+        The input array, but with all or a subset of the
+        dimensions of length 1 removed. This is always `a` itself
+        or a view into `a`.
+
+    Examples
+    --------
+    >>> x = np.array([[[0], [1], [2]]])
+    >>> x.shape
+    (1, 3, 1)
+    >>> np.squeeze(x).shape
+    (3,)
+    >>> np.squeeze(x, axis=(2,)).shape
+    (1, 3)
+
+    Notes
+    -----
+    This function differs to the original `numpy.squeeze
+    <https://docs.scipy.org/doc/numpy/reference/generated/numpy.squeeze.html>`_ in
+    the following aspects:
+
+    - The default value type is `float32` instead of `float64` in numpy.
+    - `a` only supports ndarray.
+    """
+    pass
+
+
+def _npi_ones(shape, dtype=None, order='C'):
+    r"""
+    ones(shape, dtype=None, order='C')
+    Return a new array of given shape and type, filled with ones.
+
+    Parameters
+    ----------
+    shape : int or sequence of ints
+        Shape of the new array, e.g., ``(2, 3)`` or ``2``.
+
+    dtype : data-type, optional
+        The desired data-type for the array, e.g., `int`. Default is `float32`.
+
+    order : {'C', 'F'}, optional, default: C
+        Whether to store multi-dimensional data in row-major
+        (C-style) or column-major (Fortran-style) order in
+        memory.
+
+        Not Supported yet.
+
+    Returns
+    -------
+    out : ndarray
+        Array of ones with the given shape, dtype, and order.
+
+    Examples
+    --------
+    >>> np.ones(5)
+    array([ 1.,  1.,  1.,  1.,  1.])
+    >>> np.ones((5,), dtype=int)
+    array([1, 1, 1, 1, 1])
+    >>> np.ones((2, 1))
+    array([[ 1.],
+           [ 1.]])
+    >>> s = (2,2)
+    >>> np.ones(s)
+    array([[ 1.,  1.],
+           [ 1.,  1.]])
+
+    Notes
+    -----
+    This function differs to the original `numpy.ones
+    <https://docs.scipy.org/doc/numpy/reference/generated/numpy.ones.html>`_ in
+    the following aspects:
+
+    - The default value type is `float32` instead of `float64` in numpy.
+    - `order` is not supported.
+    """
+
+
+def _npi_random_uniform(low=0.0, high=1.0, size=None):
```

Comment:
This seems to be the same case as `_npi_ones`.

Reply:
fixed